### PR TITLE
BaSP-MR-228: Profile Avatars

### PR DESCRIPTION
**Pull request by jramire5**
Now, every user has an avatar.
The avatar will be displayed in the header, before the name and last name
But it varies depending on the role;
Should let members choose an avatar from a list, when editing their own profile.
The option to choose an avatar is not available in the sign up form to keep it simple.
Trainers, admins and super-admins will have a fixed avatar for the rol. An avatar shared for admins and superadmins and another one for trainers.
Please, check this PR altogheter with this server's branch: [](https://github.com/BaSP-m2023/baru-megarocket-server/pull/117)

### CODE REVIEW

**Comment by FontanR** [comment]
maybe we should asign a default profile picture
first time member enter to their profile it shows this:
image
everything else works fine, i can edit my profile
and admin have their img too
great job!

File: public/assets/avatars/tinified.zip
Comment (francoax) [inline public/assets/avatars/tinified.zip:1]: Is it necessary this file ?? I didn't find where you are usign it

Comment (jramire5) [reply public/assets/avatars/tinified.zip:1]: You're right, i thought it was deleted

**Comment by jramire5** [comment]
> maybe we should asign a default profile picture first time member enter to their profile it shows this: image everything else works fine, i can edit my profile and admin have their img too great job!

Solved! Thank you

File: public/assets/avatars/tinified.zip
Comment (jramire5) [reply public/assets/avatars/tinified.zip:1]: Deleted!

**Review by IgnacioGalceran — APPROVED** [review]
Awesome

**Review by FontanR — APPROVED** [review]

**Pull request merged**